### DISCUSSION TITLE
how to bypass network blocks

### DISCUSSION THREAD

**Discussion by Perplexedykwhat** [3kh0/ext-remover#1401] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I don't want proxies because I want to access cloudfare-protected sites. vpns don't work

**Comment by schoolexploitkid** [comment, marked as answer]
use a dns or a mobile hotspot

**Comment by catfoolyou** [comment]
Turn on DNS over HTTPS in chrome settings

**Reply by akabutnicer** [reply]
admins typically tend to restrict this setting (or the settings page itself)

**Reply by catfoolyou** [reply]
I thought it was on a personal device or after unenrolling

**Comment by hackerman-101** [comment]
Use blobwifi
